src/ontology/mondo-ingest.Makefile

```diff
@@ -751,6 +752,16 @@ $(EXTERNAL_CONTENT_DIR)/mondo-otar-subset.robot.tsv:
 $(EXTERNAL_CONTENT_DIR)/mondo-medgen.robot.tsv:
 	wget "https://github.com/monarch-initiative/medgen/releases/latest/download/medgen-xrefs.robot.template.tsv" -O $@
 
+###### MalaCards #########
+
+# Managed in Google Sheets:
+# https://docs.google.com/spreadsheets/d/1peJ6TnobK9qI61PnCwwyoNpYZdZLWfZ1eQD1q_f7HQM/edit?gid=0#gid=0
+
+$(EXTERNAL_CONTENT_DIR)/mondo-malacards.robot.tsv:
+	wget "https://docs.google.com/spreadsheets/d/e/2PACX-1vTbstSNFfYBRNxUbbzmixE4Yi3k3QtiEpcEen8ScSOKVI6RZS4KqEFFSE0AqvHL1Hwtmi0OuC2IWeGb/pub?gid=0&single=true&output=tsv" -O $@
```

Comment:
Should take raw result from https://genecardscustomers.blob.core.windows.net/mondo/Mondo_MalaCards_v5.23.tsv instead and then turn it into a robot template.

Reply:
https://docs.google.com/spreadsheets/d/1peJ6TnobK9qI61PnCwwyoNpYZdZLWfZ1eQD1q_f7HQM/edit?gid=0#gid=0